tsconfig.json

```diff
@@ -28,7 +28,7 @@
       "strictNullChecks": true,              /* Enable strict null checks. */
       // "strictFunctionTypes": true,           /* Enable strict checking of function types. */
       // "strictBindCallApply": true,           /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
-      // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
+       "strictPropertyInitialization": false,  /* Enable strict checking of property initialization in classes. */
```

Comment:
Please keep this on true, this is the entire point of typescript. It's just bad practice to disable this.

Reply:
I was not aware this was changed. Will do!